tests/Morphir/Elm/FrontendTests.elm

```diff
@@ -24,46 +26,46 @@ frontendTest =
         sourceA =
             { path = "My/Package/A.elm"
             , content =
-                unindent """
-module My.Package.A exposing (..)
-
-import My.Package.B exposing (Bee)
-
-type Foo = Foo Bee
-
-type alias Bar = Foo
-
-type alias Rec =
-    { field1 : Foo
-    , field2 : Bar
-    , field3 : Bool
-    , field4 : Int
-    , field5 : Float
-    , field6 : String
-    , field7 : Maybe Int
-    , field8 : List Float
-    }
-                """
+                String.join "\n"
+                    [ "module My.Package.A exposing (..)"
+                    , ""
+                    , "import My.Package.B exposing (Bee)"
+                    , ""
+                    , "type Foo = Foo Bee"
+                    , ""
+                    , "type alias Bar = Foo"
+                    , ""
+                    , "type alias Rec ="
+                    , "    { field1 : Foo"
+                    , "    , field2 : Bar"
+                    , "    , field3 : Bool"
+                    , "    , field4 : Int"
+                    , "    , field5 : Float"
+                    , "    , field6 : String"
+                    , "    , field7 : Maybe Int"
+                    , "    , field8 : List Float"
+                    , "    }"
+                    ]
             }
```

Comment:
Isn't this much harder to write?

Reply:
It has advantages especially when you want to simulate string interpolation. We will rarely have large chunks of code snippets. Even this one I'm planning to break up.

Reply:
What does simulate string interpolation mean?